hack/ci/e2e-k8s.sh

```diff
@@ -161,6 +181,10 @@ EOF
     --wait=1m \
     -v=3 \
     "--config=${ARTIFACTS}/kind-config.yaml"
+
+  # Patch kube-proxy to set the verbosity level
```

Comment:
eww, we can't do this in kubeadm?
kube-proxy component config?

Reply:
I didn't see how, the `command` field is hardcoded
https://github.com/kubernetes/kubernetes/blob/9a20f30745477423e8776bb0d05316c00648f49b/cmd/kubeadm/app/phases/addons/proxy/manifests.go#L78-L81

maybe @neolit123 knows a better way

Reply:
the kube-proxy config does not expose klog verbosity. updating klog verbosity *after* a config file is parsed, means one has to do, something like: `flag.Set("v", someValue)`

i don't think any component (that uses componentConfig) supports that yet.

Reply:
And kubeadm provides no way to customize kube-proxy args?

Reply:
no, there are no plans to support that.
kubeadm might be deploying a kube-proxy operator in the future:
https://github.com/kubernetes-sigs/cluster-addons/tree/master/kubeproxy

Reply:
To clarify: I don't object to the mechanism itself, except that this should really be an initial value, and not involve a restart of all of kube-proxy. _That_ is non-ideal.

I don't think an operator will help this, and if anything will increase the core overhead, I'd love to know more about what motivates this operator.